pipeline/run_terraform.sh

```diff
@@ -1,24 +1,28 @@
+#!/bin/bash
 set -xe
 
-terraform -v
+TERRAFORM="${TERRAFORM:-terraform-0.12.29}"
 
-# unique deployment ID to avoid collisions in CI
-# needs to be 32 characters or less and start with letter
-DEPLOYMENT_ID=ci$(echo "$DRONE_REPO_NAME$DRONE_BUILD_NUMBER" | md5sum | awk '{print substr($1,0,30)}')
-echo $DEPLOYMENT_ID
+$TERRAFORM -v
 
-cp providers.tf.example examples/$EXAMPLE/providers.tf
-cp backend.tf.example examples/$EXAMPLE/backend.tf
-cd examples/$EXAMPLE
-sed -i "s/REPLACE/$DEPLOYMENT_ID/g" backend.tf
+cp providers.tf.example examples/"$EXAMPLE"/providers.tf
+cp backend.tf.example examples/"$EXAMPLE"/backend.tf
+cd examples/"$EXAMPLE"
 
-terraform init
-
-if [ $DESTROY -eq 1 ]; then
-  terraform destroy --auto-approve -var "deployment_id=$DEPLOYMENT_ID" -refresh=false -lock=false --target=module.astronomer_aws_from_scratch.module.astronomer
-  terraform destroy --auto-approve -var "deployment_id=$DEPLOYMENT_ID" -refresh=false -lock=false --target=module.astronomer_aws_from_scratch.module.system_components
-  sleep 30
-  terraform destroy --auto-approve -var "deployment_id=$DEPLOYMENT_ID" -refresh=false -lock=false
+if [ "${DESTROY:-0}" -eq 1 ]; then
+  DEPLOYMENT_ID=ci$(echo "$CIRCLE_PROJECT_REPONAME$CIRCLE_BUILD_NUM" | md5sum | awk '{print substr($1,0,30)}')
```

Comment:
IMHO it's easier to read with braces when variables are butted up against each other and have long capitalized snake case names.
```suggestion
  DEPLOYMENT_ID=ci$(echo "${CIRCLE_PROJECT_REPONAME}${CIRCLE_BUILD_NUM}" | md5sum | awk '{print substr($1,0,30)}')
```